### PR TITLE
(WIP) LKE-2456 export graph as an image 

### PR DESCRIPTION
**Pull request by aminerhanemi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+.npmrc
+node_modules
+.idea
+custom-image-export.lke/buildcache
```
Comment (francescoinfante) [inline .gitignore:5]: it's bad practice to push the whole built artifact in the codebase. wheneve you change something, you also have to change the artifact making PR unreadable.
Instead, customers should be able to build very easily with npm run build for example.

(also we are going to ship *.lke packages as releases)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "devDependencies": {
+    "@types/body-parser": "^1.19.0",
+    "@types/express": "^4.17.9",
+    "@types/superagent": "^4.1.10",
+    "browserify": "^17.0.0",
+    "express": "^4.17.1",
+    "npm-run-all": "^4.1.5",
+    "typescript": "^4.1.3"
+  },
+  "dependencies": {
```
Comment (francescoinfante) [inline package.json:32]: how do we put ogma in here?

File: src/backend/routeHandler.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      } else {
+        res.status(412);
+        res.send(JSON.stringify({status: 412, body: {message: LKEConfiguration.body.message}}));
+      }
+    } catch (e) {
+      res.status(412);
+      res.send(JSON.stringify({status: 412, body: e}));
+    }
+  });
+  options.router.get('/getVisualizationConfiguration/:id&:sourceKey', async (req, res) => {
```
Comment (francescoinfante) [inline src/backend/routeHandler.ts:23]: this is a very non standard way to get parameters, instead, let's pass them as querystring.

File: custom-image-export.lke/public/index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html class="no-js" lang="">
```
Comment (francescoinfante) [inline custom-image-export.lke/public/index.html:2]: We should create a separate folder for the assets instead of the *.lke itself directly

**Review by francescoinfante — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (aminerhanemi) [reply package.json:32]: ogma is part of ogma-helper

**Review by francescoinfante — DISMISSED** [review]

**Pull request merged**